cf/cognito-case-insensitive.yaml

```diff
@@ -24,8 +24,120 @@ Outputs:
     Value: !GetAtt UserPool.ProviderName
     Export:
       Name: !Sub 'biomage-user-pool-case-insensitive-${Environment}::UserPoolProviderName'
+  UIRoleArn:
+    Description: ARN of the UI role
+    Value: !GetAtt UIRole.Arn
+    Export:
+      Name: !Sub 'biomage-user-pool-case-insensitive-${Environment}::UIRoleArn'
+
+Resources: 
+  UIRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: !Sub "biomage-user-pool-case-insensitive-auth-user-role-${Environment}"
```

Comment:
rename to `"biomage-user-pool-case-insensitive-auth-user-role-${Environment}"` so it will be clear that it's a role for the UI